packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js

```diff
@@ -0,0 +1,3 @@
+export default class AccountUserIdentityDocumentsController {
+  /* @ngInject */
+}
```

Comment:
Empty controller can be delete.

Reply:
The scope of this PR is to set-up an empty page. We have a different ticket to complete the page.

Reply:
Ah ok, on my side i have habit to let a todo comment, to inform others about the incoming tasks.
```suggestion
/**TODO: controller will be completed within incoming tasks*/
export default class AccountUserIdentityDocumentsController {
  /* @ngInject */
}
```